docs/mutations.md

```diff
@@ -596,28 +584,43 @@ contains nonzero entries, then the mutated tree sequence will
 (probably) contain silent mutations.
 
 Some facts about Markov chains are useful to interpret the statistics
-of these mutation models.
-First, suppose we have tabulated all mutations, and so for each pair of alleles
-{math}`i` and {math}`j` we have the proportion of mutations that caused an {math}`i \to j` change.
-If allele {math}`i` mutates to a different allele, the chance it mutates to allele {math}`j`
-is proportional to `transition_matrix[i,j]` but excluding the diagonal (no-change) entry,
-so is equal to `transition_matrix[i,j] / (1 - transition_matrix[i,i])`.
+of these mutation models. First, suppose we have tabulated all mutations, and so
+for each pair of alleles {math}`i` and {math}`j` we have the proportion of mutations
+that caused an {math}`i \to j` change. If allele {math}`i` mutates to a different allele,
+the chance it mutates to allele {math}`j` is proportional to `transition_matrix[i,j]` but
+excluding the diagonal (no-change) entry, so is equal to
+`transition_matrix[i,j] / (1 - transition_matrix[i,i])`.
 Second, suppose that an ancestor carries allele {math}`i` at a given position.
-The probability that her descendant some time {math}`t` in the future carries allele {math}`j` at that position
-is given by a matrix exponential of
-the scaled [infinitesimal rate matrix](<https://en.wikipedia.org/wiki/Transition_rate_matrix>) of the Markov chain,
-which can be computed as follows:
+The probability {math}`P(t)` that her descendant some time {math}`t` in the future
```

Comment:
We shouldn't introduce new notation (here, P(t)) if we don't actually use the notation anywhere. Perhaps add the equation (in math) that P(t) = e^{tQ}?

Reply:
whoops that's P(t) = e^{t r Q}, where r is the rate